apps/oauth2/lib/Settings/Admin.php

```diff
@@ -22,12 +22,38 @@
 
 namespace OCA\OAuth2\Settings;
 
+use OC\InitialStateService;
+use OCA\OAuth2\Db\ClientMapper;
 use OCP\AppFramework\Http\TemplateResponse;
 use OCP\Settings\ISettings;
 
 class Admin implements ISettings {
 
+	/** @var InitialStateService */
+	private $initialStateService;
+	/** @var ClientMapper */
+	private $clientMapper;
+
+	public function __construct(InitialStateService $initialStateService, ClientMapper $clientMapper) {
```

Comment:
```suggestion
	public function __construct(IInitialStateService $initialStateService, ClientMapper $clientMapper) {
```
Shouldn't this be IInitialStateService ?

Reply:
Yes it should